wp-content/themes/core/components/blocks/gallery_grid/css/gallery_grid.pcss

```diff
@@ -0,0 +1,88 @@
+.b-gallery-grid__content {
+    padding-top: var(--spacer-40);
+}
+
+.b-gallery-grid__header {
+
+	@media (--viewport-full) {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+	}
+
+	.b-gallery-grid__cta {
+		margin-top: var(--spacer-30);
+		min-width: 250px;
+
+		@media (--viewport-full) {
+			text-align: right;
+		}
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Gallery Grid
+ *
+ * ----------------------------------------------------------------------------- */
+
+.gallery-layout--one {
```

Comment:
Please go ahead and use CSS grid (and grid gutters) for these layouts. No need to use flexbox for grid layouts. :tada: